jest/setupTests.ts

```diff
@@ -3,8 +3,14 @@ import {configure} from 'enzyme';
 import {toHaveNoViolations} from 'jest-axe';
 import serializer, {matchers} from 'jest-emotion';
 import '@testing-library/jest-dom/extend-expect';
+import {verifyComponent} from './verifyComponent';
+import {jest} from '@jest/globals';
 
 expect.addSnapshotSerializer(serializer);
 expect.extend(toHaveNoViolations);
 expect.extend(matchers);
 configure({adapter: new Adapter()});
+
+// add convenience variables to the global context
+(global as any).verifyComponent = verifyComponent;
+(global as any).jest = jest;
```

Comment:
I didn't need to add `jest`, but newer jest versions seemed to drop `jest` as an automatic global.